src/systemtest/java/util/PropertiesHandler.java

```diff
@@ -0,0 +1,39 @@
+package util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map.Entry;
+import java.util.Properties;
+
+public class PropertiesHandler {
+
+    public static void setProperties() throws Throwable {
+        final String systemTestFilePath;
+
+        if (System.getProperty("propertiesFile") != null) {
+            systemTestFilePath = System.getProperty("propertiesFile");
+        } else {
+            systemTestFilePath = "src/test/resources/system_test.properties";
+        }
+        InputStream systemTestFileInputStream = new FileInputStream(new File(systemTestFilePath));
+        Properties systemTestProperties = new Properties();
+        systemTestProperties.load(systemTestFileInputStream);
+        for (Entry entry : systemTestProperties.entrySet()) {
+            String key = entry.getKey()
+                              .toString();
```

Comment:
is toString needed for Entry key and values? they "should" already be String.

Reply:
It depends on the key value, getKey() return key value as it is..